addon/i3dio/ui/object.py

```diff
@@ -872,6 +893,42 @@ def draw(self, context):
         row.prop(obj.i3d_mapping, 'mapping_name')
 
 
+@persistent
+def check_lod_distance(dummy):
+    for obj in bpy.data.objects:
+        if obj.type == 'EMPTY':
+            if hasattr(obj, 'i3d_attributes') and 'lod_distance' in obj['i3d_attributes']:
+                current_lod = obj['i3d_attributes']['lod_distance']
+                # Skip if already converted
+                if not isinstance(current_lod, str):
+                    continue
+
+                # Check for placeholder or empty strings
+                if current_lod in ('', 'Enter your LOD Distances if needed.'):
+                    print(f"LOD distance for {obj.name} is not set.")
+                    continue
+
+                try:
+                    # Convert from string to list of floats
+                    lod_distance_str = current_lod.split()
+                    new_lod_distance = [float(x) for x in lod_distance_str]
+                    new_lod_distance = (new_lod_distance + [0.0] * 4)[:4]
+
+                    # first value should always be 0.0
+                    new_lod_distance[0] = 0.0

```

Comment:
The new LOD distance could be initialised from the default value
```suggestion
                    new_lod_distance = I3DNodeObjectAttributes.i3d_map['lod_distance']['default']
```